modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Removing content from a CR in managed clusters
+
+You can remove content from a custom resource (CR) deployed to a managed cluster through a policy.
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `"musthave"`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
+A `"musthave"` policy without the removed field is still compliant because the CR on the managed cluster has all the other content.
+
+To remove the content from the managed cluster, set the `complianceType` field to `"mustonlyhave"`.
+In this mode, the policy ensures that the CR on the cluster is an exact match of what is specified in the policy.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Remove the content that you no longer need from the affected CRs.
+
+. Change the `complianceType` of the affected policies to `"mustonlyhave"`.
+
+. Commit and push to the changes to the remote repository.
+
+. Wait for the changes to synchronize through the application and the affected CRs to be updated on the managed clusters.
```

Comment:
If TALM is in use the user is generating inform policies and will need to enforce the policy through use of TALM.